packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -21,6 +22,7 @@ const log = createLogger('xo:xo-mixins:metadata-backups')
 const DIR_XO_CONFIG_BACKUPS = 'xo-config-backups'
 const DIR_XO_POOL_METADATA_BACKUPS = 'xo-pool-metadata-backups'
 const METADATA_BACKUP_JOB_TYPE = 'metadataBackup'
+const POOL_METADATA_TIMEOUT = 1e4
```

Comment:
**No.**

I don't want static values in the code, move it to the configuration.